EMacco/Customer-Ticketing-System
# #168825026 Login Page

#### What does this PR do?
This PR allows a user to be able to login to the system and stay logged in

#### Description of Task to be completed?
- Field for email
- Field for password
- Submit button
- Stay logged in

#### How should this be manually tested?
- Checkout to this branch
- Start the servers using `foreman start`
- Navigate to `http://localhost:3000/auth` on your browser
- Enter your email and password
- You should be validated and logged in

#### What are the relevant pivotal tracker stories?
[#168825026](https://www.pivotaltracker.com/story/show/168825026)

#### Screenshots (if appropriate)
![login](https://user-images.githubusercontent.com/20377428/65833130-9bfc9580-e2c4-11e9-9b3f-15702d5a7b17.PNG)
